apple/Handlers/RNPanHandler.m

```diff
@@ -78,75 +87,142 @@ - (void)activateAfterLongPress
   [_gestureHandler handleGesture:self inState:RNGestureHandlerStateActive];
 }
 
-- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
+- (void)interactionsBegan:(NSSet *)touches withEvent:(UIEvent*)event
 {
-  if ([self numberOfTouches] == 0) {
-    [_gestureHandler reset];
-  }
-#if !TARGET_OS_TV
-  if (_hasCustomActivationCriteria) {
-    // We use "minimumNumberOfTouches" property to prevent pan handler from recognizing
-    // the gesture too early before we are sure that all criteria (e.g. minimum distance
-    // etc. are met)
-    super.minimumNumberOfTouches = 20;
-  } else {
-    super.minimumNumberOfTouches = _realMinimumNumberOfTouches;
-  }
-#endif
-  [super touchesBegan:touches withEvent:event];
-  [_gestureHandler.pointerTracker touchesBegan:touches withEvent:event];
+    [_gestureHandler.pointerTracker touchesBegan:touches withEvent:event];
+
+    if (touches.count == 0) {
+      [_gestureHandler reset];
+    }
+
+  #if !TARGET_OS_TV && !TARGET_OS_OSX
+    if (_hasCustomActivationCriteria) {
+      // We use "minimumNumberOfTouches" property to prevent pan handler from recognizing
+      // the gesture too early before we are sure that all criteria (e.g. minimum distance
+      // etc. are met)
+      super.minimumNumberOfTouches = 20;
+    } else {
+      super.minimumNumberOfTouches = _realMinimumNumberOfTouches;
+    }
+  #endif
+
   [self triggerAction];
 
   if (!isnan(_activateAfterLongPress)) {
     [self performSelector:@selector(activateAfterLongPress) withObject:nil afterDelay:_activateAfterLongPress];
   }
 }
 
-- (void)touchesMoved:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
+- (void)interactionsMoved:(NSSet *)touches withEvent:(UIEvent*)event
 {
-  [super touchesMoved:touches withEvent:event];
```

Comment:
Same here, why do we remove `super` call?

Reply:
same as above